Tests/Crypto/KeypairDeriviationTests.swift

```diff
@@ -16,9 +16,9 @@ class KeypairDeriviationTests: XCTestCase {
         try performTest(filename: "ecdsaHDKD", keypairFactory: EcdsaKeypairFactory())
     }
 
-    private func performTest(filename: String, keypairFactory: KeypairFactoryProtocol) throws {
+    private func performTest(filename: String, keypairFactory: KeypairFactoryProtocol, useMiniSeed: Bool = true) throws {
```

Comment:
Why do we need `useMiniSeed` if it is not used?

Reply:
Well, it's there if someone (us?) would want to use it (: 
I needed it first for BIP32 tests then I decided to separate them, so I thought it might be a good idea to leave it there for future uses

Reply:
Let's remove it for now if it is not used

Reply:
Dine